### ISSUE TITLE
fetchAllSpaces & fetchAllCollections kills browser on mobile devices

### ISSUE THREAD

**Issue by adam-635** [buildcore-io/core#1402]
Currently "fetchAllSpaces" and "fetchAllCollections" does listen on all records in the database. This is very inefficient and kills the browser. We must use individual retrieval instead. This should not be "continuous" listen but only once off. No streaming of updates.

Change on all places where fetchAllSpaces & fetchAllCollections is used to instead pass collection IDs to retrieve and cache them.

notes:
- Remove on member activity page as it's no longer needed.

Some of the issues:
- open filters on iPhone, it kills the browser
- space about, kills the browser
